quickstart.md

```diff
@@ -40,16 +40,34 @@ Minimal example:
 loadGM(initCanvas, { version: 'latest' });
 
 function initCanvas() {
-  canvas = new gmath.Canvas('#gm-div');
+  const canvasOptions = {
+    // You may use options described on
+    // https://github.com/eweitnauer/gm-api/blob/master/customizing-gm-embedded-as-an-iframe.md#options-that-go-in-the-options-option-described-above
+  }
+  canvas = new gmath.Canvas('#gm-div', canvasOptions);
   canvas.model.createElement('derivation', { eq: '2x+1=3', pos: { x: 'center', y: 50 } });
 }
 </script>
 </body>
 ```
 
+<!-- If you change the name of this section, there is at least one place in the graspable-math repository that has a link to this section (displayed on GitHub), so you will need to update that(those) hyperlink(s).  -->
 ## Loading and Saving a Canvas State
 
-Use the methods `canvas.toJSON()` and `canvas.loadFromJSON()` to save and load canvas states.
+### If you want users to use our built-in Save button
+
+That should work by default. The canvases will be saved on our servers.
+
+### If you plan on storing saved canvases yourself
+
+Then you must set the `"use_built_in_saving_backend"` and `"save_btn"` canvas options to `false`.
+(To set canvas options, see [this section](https://github.com/eweitnauer/gm-api/blob/master/quickstart.md#embedding-a-graspable-math-canvas) or [this page](https://github.com/eweitnauer/gm-api/blob/master/customizing-gm-embedded-as-an-iframe.md) depending on your situation.)
+If you do not head our warning to set `"use_built_in_saving_backend"` to `false`, then images on the canvas will be uploaded to servers we control,
+and, instead of storing the images as data URLs along with the rest of the canvas, only links to the uploaded images will be stored in the canvas.
+This makes the file size of a canvas containing an image orders of magnitude smaller, probably.
```

Comment:
```suggestion
This can reduce the file size of a canvas containing images by orders of magnitude.
```

Reply:
Nope It can't always reduce the file size of a canvas containing images by orders of magnitude. What if the images were super small, and there was a ton of not-images in the file? So that suggestion is innacureate

Reply:
Ok. I sumbitted a change.